src/components/FilterChecklist.vue

```diff
@@ -2,12 +2,11 @@
   <fieldset>
     <div class="filters-title" @click.prevent="toggleFilterVisibility">
       <span>{{ title }}</span>
-
-      <button class="filter-dropdown-visibility-button">
```

Comment:
Please fix indentations here. Your button opening tag is a couple levels behind the closing, and the img tag should be one more level in. You can also break the img element down into multiple lines like so: 
```
<img src="@/assets/arrow.svg"
  v-bind:class="{rotImg:filtersVisible}"
  alt="toggle filters visibility"/>
```
This avoids the eslint warning about line length, and avoids disabling all eslint for that line.

Reply:
@akmadian  Changes done !!